src/templates/finops-hub/schema/schema_ea_normalized.json

```
@@ -0,0 +1,723 @@
+      "value": 1.0
+    },
+    {
+      "name": "invoiceId",
+      "value": ""
+    },
+    {
+      "name": "location",
+      "value": "$$COLUMN:ResourceLocation"
+    },
+    {
+      "name": "paygCostInBillingCurrency",
+      "value": "$$COLUMN:BillingCurrencyCode"
+    },
+    {
+      "name": "paygCostInUsd",
+      "value": 0.0
+    },
+    {
+      "name": "previousInvoiceId",
+      "value": ""
+    },
+    {
+      "name": "pricingCurrency",
+      "value": "$$COLUMN:BillingCurrencyCode"
+    },
+    {
+      "name": "provider",
+      "value": "azure"
+    },
+    {
+      "name": "publisherId",
+      "value": ""
+    },
+    {
+      "name": "resellerMpnId",
+      "value": ""
+    },
+    {
+      "name": "resellerName",
+      "value": ""
+    },
+    {
+      "name": "servicePeriodEndDate",
+      "value": "$$COLUMN:BillingPeriodEndDate"
+    },
+    {
+      "name": "servicePeriodStartDate",
+      "value": "$$COLUMN:BillingPeriodStartDate"
+    }
+  ],  
+  "translator": {
+    "type": "TabularTranslator",
+    "mappings": [
+      {
+        "source": {
+          "name": "AdditionalInfo",
+          "type": "String"
+        },
+        "sink": {
+          "name": "AdditionalInfo",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "benefitId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "BenefitId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "benefitName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "BenefitName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "BillingAccountId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "BillingAccountId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "BillingAccountName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "BillingAccountName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "BillingCurrencyCode",
+          "type": "String"
+        },
+        "sink": {
+          "name": "BillingCurrency",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "BillingPeriodEndDate",
+          "type": "DateTime"
+        },
+        "sink": {
+          "name": "BillingPeriodEndDate",
+          "type": "DateTime"
+        }
+      },
+      {
+        "source": {
+          "name": "BillingPeriodStartDate",
+          "type": "DateTime"
+        },
+        "sink": {
+          "name": "BillingPeriodStartDate",
+          "type": "DateTime"
+        }
+      },
+      {
+        "source": {
+          "name": "AccountOwnerId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "BillingProfileId",
```

AccountOwnerId is not the same as a billing profile. That needs to be kept in the normalized schema. There's nothing to map it to in MCA.
```suggestion
          "name": "AccountOwnerId",
```

---

Done